[DOCS] Add session ID to highlighted fields section in alert details flyout (#2067)
elastic/security-docs: docs/detections/alerts-ui-manage.asciidoc

The *Highlighted Fields* section displays the most relevant fields for the alert type. Use this section to inform your triage efforts as you investigate the alert.

> NOTE: The *Session ID* field provides a unique ID for the Linux session. To collect it and other session data, you must enable the *Include session data* setting on your Endpoint Security integration policy. Refer to <<enable-session-view, Enable Session View data>> for more information.

Reviewer: "the Linux session" is a little awkward since otherwise this doesn't seem to be talking about a specific session. Alternatively, something like "the Linux session that generated the alert" but that's a bit wordy.

Also li'l shared attribute fix:
```suggestion
NOTE: The *Session ID* field provides a unique ID for Linux sessions. To collect it and other session data, you must enable the *Include session data* setting on your {endpoint-cloud-sec} integration policy. Refer to <<enable-session-view, Enable Session View data>> for more information.
```

> NOTE: The *Session ID* field provides a unique ID for Linux sessions. To collect it and other session data, you must enable the *Include session data* setting on your {endpoint-cloud-sec} integration policy. Refer to <<enable-session-view, Enable Session View data>> for more information.

Author: Yeah, I think it's worth noting. I was actually trying to remember what the field name was so thank you for providing it here!

> NOTE: The *Session ID* field shows the number of alerts generated on a Linux host in a session. To begin collecting session data, you must enable the *Include session data* setting on your Endpoint Security integration policy. Refer to <<enable-session-view, Enable Session View data>> for more information.

Reviewer 2: ```suggestion
NOTE: The *Session ID* field provides a unique ID for the Linux session. To collect it and other session data, you must enable the *Include session data* setting on your Endpoint Security integration policy. Refer to <<enable-session-view, Enable Session View data>> for more information.
```

> The *Alert prevalence* column shows the total number of alerts within the selected timeframe that have identical values. For example, an alert with an alert prevalence of 3 for the `host.name` field means three alerts with the same `host.name` value exist within the given timeframe. Alert prevalence data can help you investigate relationships with other alerts and gain more context about the event producing the alert.

Reviewer 3: It might be worth noting here that users can click on the alert prevalence count in order to investigate the alerts in timeline.

Reviewer 3: Ah sorry, I did not see that one :D Thanks so much!
